persistence/azure-table/configuration_code_astp_[5,).partial.md

```diff
@@ -0,0 +1,96 @@
+For sagas and subscriptions:
+
+snippet: AzurePersistenceAllConnectionsCustomization
+
+### Saga configuration
+
+snippet: AzurePersistenceSagasCustomization
+
+The following settings are available for changing the behavior of saga persistence section:
+
+ * `ConnectionString`: Sets the connection string for the storage account to be used for storing saga information.
+ * `UseTableServiceClient`: Allows to set a fully pre-configured Table Service client instead of using a connection string.
+
+#### Saga compatibility configuration
+
+snippet: AzurePersistenceSagasCompatibility
+
+The following settings are available for changing the behavior of saga persistence compatibility section:
+
+ * `EnableSecondaryKeyLookupForSagasCorrelatedByProperties`: To enable the compatibility mode and force the persister to find sagas by the secondary index property. This is only useful while there are still sagas that use the secondary index property `NServiceBus_2ndIndexKey`.
+ * `AssumeSecondaryKeyUsesANonEmptyRowKeySetToThePartitionKey`: Sagas that have been stored with a secondary index used an empty RowKey on the secondary index entry. By enabling this setting the secondary key lookups will assume that the RowKey equals the PartitionKey, which is crucial for Azure Cosmos DB Table API usage.
+ * `AllowSecondaryKeyLookupToFallbackToFullTableScan`: Opt-in to full table scanning for sagas that have been stored with version 1.4 or earlier when running in compatibility mode.
+
+### Subscription configuration
+
+snippet: AzurePersistenceSubscriptionsCustomization
+
+The following settings are available for changing the behavior of subscription persistence:
+
+ * `ConnectionString`: Sets the connection string for the storage account to be used for storing subscription information.
+ * `UseTableServiceClient`: Allows to set a fully pre-configured Table Service client instead of using a connection string.
+
+### Configuring a Table Service Client Provider
+
+A fully preconfigured TableServiceClient can be registered in the container through a custom provider.
```

Comment:
I see a mix of Table Service Client versus TableServiceClient. Should we align or does the mix follow a pattern? Given this is a specific type should we use the type indicator TableServiceClient?